do-catalog/src/styles/components/_buttons.scss

```diff
@@ -0,0 +1,112 @@
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 10px 16px;
+  border-radius: 4px;
+  background: $button__bg-color;
+  color: $button__color;
+  font: 600 12px/16px "Montserrat", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  letter-spacing: 0;
+  cursor: pointer;
+
+  &:focus {
+    outline: none;
+  }
+
+  &:hover,
+  &:focus {
+    background: $button__bg-color--hover;
+  }
+}
+
+a {
+  &.button:hover {
+    text-decoration: none;
+  }
+}
+
+.button--ghost {
+  background: transparent;
+  color: $button-ghost__color;
+
+  &:hover,
+  &:focus {
+    background: transparent;
+    text-decoration: underline;
+  }
+}
+
+.button--outline {
+  border: 1px solid $button-outline__border-color;
+  background: $button-outline__bg-color;
+  color: $button-outline__color;
+
+  &:hover,
+  &:focus {
+    background: $button-outline__bg-color--hover;
+  }
+}
+
+.button--cta {
+  padding-top: 1em;
+  padding-bottom: 1em;
+  background: $button-cta__bg-color;
+  color: $button-cta__color;
+  font-size: 16px;
+
+  &:hover,
+  &:focus {
+    background: $button-cta__bg-color--hover;
+  }
+}
+
+.button--small {
+  padding: 8px 12px;
+  font: 600 10px/14px "Montserrat", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  text-transform: uppercase;
+}
+
+.button--alert {
+  background: $button-alert__bg-color;
+
+  &:hover,
+  &:focus {
+    background: $button-alert__bg-color--hover;
+  }
+}
+
+.button-icon {
+  width: 14px;
+  height: 14px;
+  margin-right: 8px;
+}
+
+.button--last {
+  padding-right: 0;
+}
+
+.buttonLink {
+  display: inline-block;
+  overflow: hidden;
+  font: 600 12px/16px "Montserrat", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  letter-spacing: 0;
+  text-transform: uppercase;
+
+  svg {
+    margin-right: 8px;
+    transform: translate3d(0, 0, 0);
+    transition: transform 300ms ease-in;
+  }
+
+  &:hover {
+    svg {
+      transform: translate3d(4px, 0, 0);
```

Comment:
Why `translate3d` for just one axis? Any performance improvement?

Reply:
No idea to be honest, it's imported from the dashboard. I know however sometimes people use the translate3d to improve performance by using gpu, but no idea in this particular case